doc/write_ex.md

```diff
@@ -0,0 +1,147 @@
+# How to write an exercise?
+
+Every input file begins with the required library calls, needed by the test
+function.
+
+```ocaml
+open Test_lib
+open Report
+```
+
+## Prelude and prepare
+
+Prelude and prepare functions are written almost as-if. You just write your
```

Comment:
Please rephrase "Prelude and prepare functions". It does not make sense for someone that forgot the generated filenames.